.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - 3.5
+
+install:
+  - pip install Django
```

Comment:
If we support multiple versions of Django we would have to install all the different versions of Django and run our test suite once for each version.

I don't think we need to support different versions for the scope of this project.

Although I do think that we should specify which version to install in this script, I'll add that